static/src/stylesheets/module/content/_recipe-article.scss

```diff
@@ -103,34 +101,122 @@ $padding-wide: 100px;
 }
 
 .recipe__label {
+    @include f-headline;
     position: absolute;
     z-index: 300;
-    background-color: $purple;
-    height: 90px;
-    width: 160px;
+    background-color: $features-main-2;
+    height: 65px;
+    width: 110px;
     border-radius: 50%;
     text-align: center;
-    padding-top: 70px;
+    padding-top: 45px;
     color: #ffffff;
-    left: 1px;
-    top: 0;
+    left: -$gs-gutter / 6;
+    bottom: -$gs-gutter * 1.6;
     line-height: initial;
+    font-weight: 900;
 
-    @include mq($from: tablet) {
-        top: 5px;
-        left: 20px;
+    @include mq($from: desktop) {
+        top: -$gs-gutter * 2;
+        left: 14%;
+        bottom: auto;
     }
 }
 
 .recipe__article {
-    padding: 0 12px;
-    padding-bottom: 44px;
+    padding: 0 $gs-gutter / 2;
+    padding-bottom: $gs-gutter * 4;
+}
+
+.recipe__gutter-wrapper {
+    display: none;
+
+    @include mq($from: desktop) {
+        display: flex;
+        flex-basis: 5%;
+        position: sticky;
+        top: 0;
+        height: 100vh;
```

Comment:
Should we be a bit more careful here, since we say we'd support older browsers? I'm not 100% sure ...